utils/referrer-link-util.ts

```diff
@@ -0,0 +1,5 @@
+const safeReferrerLinkRegex = /^(\/[a-zA-Z0-9]+)+$/
+
+export const isSafeReferrerLink = (referrerLink: string) => {
+  return safeReferrerLinkRegex.test(referrerLink);
+}
```

Comment:
This is to prevent people from sending other people links like `https://serotracker-git-issue-118-clicki-abe9e2-himanshurankas-projects.vercel.app/pathogen/arbovirus/visualizations?visualization=median-seroprevalence-by-who-region&referrerRoute=https://airtable.com/` Where `https://airtable.com/` is a link to some kind of malicious website that you want them to visit when they click back. All I'm doing here is guaranteeing that we're going to somewhere internal.